mlflow/tracking/service.py

```diff
@@ -97,6 +98,12 @@ def log_param(self, run_id, key, value):
         param = Param(key, str(value))
         self.store.log_param(run_id, param)
 
+    def set_tag(self, run_id, key, value):
+        """Sets a tag on the given run id. Value will be converted to a string."""
+        _validate_tag_name(key)
+        tag = RunTag(key, str(value))
+        self.store.set_tag(run_id, tag)
```

Comment:
Cover the case of overwriting tags for a run. And how do we want to handle that?

Reply:
Added in test_file_store